site/content/en/docs/Integration Patterns/high-density-gameservers.md

```diff
@@ -6,7 +6,7 @@ weight: 70
 description: >
   How to run multiple concurrent game sessions in a single GameServer process.
 ---
-{{< beta title="Allocation State Filter" gate="StateAllocationFilter" >}}
+{{< stable title="Allocation State Filter" gate="StateAllocationFilter" >}}
```

Comment:
Here we can just take the line away - they can't disable the gate anyways.

Reminder, though, that docs changes need a release guard: https://agones.dev/site/docs/contribute/#within-a-page